### DISCUSSION TITLE
Bus error (core dumped) when running rgnn.py

### DISCUSSION THREAD

**Discussion by luishengjie** [snap-stanford/ogb#209] [MAG240M-LSC]
I have the following error when trying to run the rgnn baseline model. I have a 128GB RAM and a nvidia tesla v100 GPU.
**Error Message:**
GPU available: True, used: True
TPU available: False, using: 0 TPU cores
Merging adjacency matrices... Done! [757.61s]
Generating full feature matrix...
Copying paper features...
 23%|████████▍                            | 279/1218 [04:04<16:04,  1.03s/it]Bus error (core dumped)

**Comment by weihua916** [comment]
Not sure about the bug error. might be because of limited CPU memory. 
Besides,  you can download pre-processed feature at https://github.com/dmlc/dgl/tree/master/examples/pytorch/ogb_lsc/MAG240M
